### PR TITLE
Gdr 2371

### PR DESCRIPTION
**Pull request by gladkia**
See: https://github.com/gdrplatform/gDRstyle/pull/71
Preview: https://gdrplatform.github.io/gDRutils/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,10 +10,11 @@ ARG GITHUB_TOKEN
 RUN mkdir -p /mnt/vol
 COPY rplatform/dependencies.yaml rplatform/.github_access_token.txt* /mnt/vol
 RUN echo "$GITHUB_TOKEN" >> /mnt/vol/.github_access_token.txt
+RUN Rscript -e 'BiocManager::install(c("BiocStyle", "qs"))'
```
Comment (gladkia) [inline Dockerfile:13]: packages in the Suggests that are essential to build vignette (which happens within `pkgdown::build_site`.

**Review by bczech — APPROVED** [review]

File: .github/workflows/pkgdown.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    branches: [main, master, test_ci_pkgdown]
+
+jobs:
+  trigger_pkgdown:
+    uses: gdrplatform/gDRstyle/.github/workflows/pkgdown.yaml@GDR-2371_2
+    with:
+      USERNAME: ${{ github.actor }}
+      PACKAGE_NAME: ${{ github.event.repository.name }}
+      BRANCH_NAME: ${{ github.head_ref }}
+      DEFAULT_BRANCH: master
```
Comment (marcin-kam) [inline .github/workflows/pkgdown.yaml:22]: Please remove also there if you remove in gDRstyle

File: .github/workflows/pkgdown.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+name: pkgdown
+
+on:
+  push:
+    branches: [main, master, test_ci_pkgdown]
+
+jobs:
+  trigger_pkgdown:
+    uses: gdrplatform/gDRstyle/.github/workflows/pkgdown.yaml@GDR-2371_2
```
Comment (marcin-kam) [inline .github/workflows/pkgdown.yaml:17]: Don't forget to change branch after gDRstyle merge

**Review by marcin-kam — APPROVED** [review]

File: .github/workflows/pkgdown.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# See https://github.com/r-lib/actions/tree/master/examples#readme for
+# additional example workflows available for the R community.
```
[the remark this answers is not in the document]
Comment (gladkia) [reply .github/workflows/pkgdown.yaml:7]: Done.

File: .github/workflows/pkgdown.yaml (hunk 2 above)
Comment (gladkia) [reply .github/workflows/pkgdown.yaml]: done.

File: .github/workflows/pkgdown.yaml (hunk 3 above)
Comment (gladkia) [reply .github/workflows/pkgdown.yaml]: done.

**Pull request merged**